src/test/kotlin/uk/gov/dluhc/printapi/service/PrintResponseFileServiceTest.kt

```diff
@@ -48,4 +53,70 @@ internal class PrintResponseFileServiceTest {
         inOrder.verify(printResponseProcessingService).processBatchAndPrintResponses(expectedPrintResponses)
         inOrder.verify(sftpService).removeFileFromOutBoundDirectory(directory, fileName)
     }
+
+    @Test
+    fun `should log and not throw exception if file is not found when deleting`() {
+        // Given
+        val directory = "EROP/Dev/OutBound"
+        val fileName = "status-20221101171156056.json"
+        val fileContent = "{\"BatchResponses\": [], \"PrintResponses\": []}"
+
+        given(sftpService.fetchFileFromOutBoundDirectory(any(), any())).willReturn(fileContent)
+        val expectedPrintResponses = PrintResponses().withBatchResponses(emptyList()).withPrintResponses(emptyList())
+        given(objectMapper.readValue(fileContent, PrintResponses::class.java))
+            .willReturn(expectedPrintResponses)
+        val exception =
+            IOException("Failed to remove file", SftpException(ChannelSftp.SSH_FX_NO_SUCH_FILE, "No such file"))
```

Comment:
I'd be happy for this exception to be the relevant MessagingException if there is an easy/convenient constructor. It would be nice if the exception thrown in the test matched the real thing?

Reply:
@NathanRussellValtech , this is the one thrown from the `SftpService`, which we agreed to change to throw `IOException`. 

It was `MessagingException` with my initial PR, but since we made the mentioned change to the `SftpService`, I made the change here to match the corresponding behavior.